src/Sylius/Bundle/AdminBundle/Action/Account/RequestPasswordResetAction.php

```diff
@@ -42,24 +45,30 @@ public function __invoke(Request $request): RedirectResponse
             /** @var PasswordResetRequest $formData */
             $formData = $form->getData();
             $requestPasswordResetMessage = new RequestResetPasswordEmail(
-                $formData->getEmail()
+                $formData->getEmail(),
             );
 
             $this->messageBus->dispatch($requestPasswordResetMessage);
-        }
 
-        $this->flashBag->set('success', 'sylius.admin.request_reset_password.success');
+            $this->flashBag->set('success', 'sylius.admin.request_reset_password.success');
+
+            $options = $request->attributes->get('_sylius');
+            $redirectRoute = $options['redirect'] ?? 'sylius_admin_login';
 
-        $options = $request->attributes->get('_sylius');
-        $redirectRoute = $options['redirect'] ?? 'sylius_admin_login';
+            if (is_array($redirectRoute)) {
+                return new RedirectResponse($this->router->generate(
+                    $redirectRoute['route'] ?? 'sylius_admin_login',
+                    $redirectRoute['params'] ?? [],
+                ));
+            }
 
-        if (is_array($redirectRoute)) {
-            return new RedirectResponse($this->router->generate(
-                $redirectRoute['route'] ?? 'sylius_admin_login',
-                $redirectRoute['params'] ?? []
-            ));
+            return new RedirectResponse($this->router->generate($redirectRoute));
         }
 
-        return new RedirectResponse($this->router->generate($redirectRoute));
+        return new Response(
+            $this->twig->render('@SyliusAdmin/Security/requestPasswordReset.html.twig', [
+                'form' => $form->createView(),
+            ])
+        );
```

Comment:
Why did you change the previous behavior 🤔?

Reply:
To render the validation errors. Pretty sure you can't do that with a redirect. I could just put them in an error flash, but that seems messy.